strings.po

```diff
@@ -21184,6 +21192,10 @@ msgid ""
 "Please free at least 15 MB to give your saves some room to breathe.\n"
 "{0}"
 msgstr ""
+"Es ist nicht genügend Speicherplatz vorhanden, um in dein Sicherungsverzeichnis zu schreiben.\n"
+"\n"
+"Bitte gib mindestens 15 MB frei für deine Sicherungen.\n"
```

Comment:
_Saves_ sind doch Speicherstände / Spielstände (ich meine das wurde auch schon ein paar mal übersetzt beim Lademenü.

Reply:
Stimmt.
Speicherstand und Spielstand werden beide verwendet.
Spielstand bzw Spielstände kommt mir hier passender vor.

Reply:
Please free at least 15 MB to give your saves some room to breathe.

nicht schlecht die speicherstände von Oxygen not Included brauchen platz zum atmen^^